packages/api/src/utils/pin.js

```diff
@@ -97,9 +109,9 @@ export function toPins (peerMap) {
  * @param {number} checkInterval
  * @return {Promise.<import('@web3-storage/db/db-client-types').PinUpsertInput[]>}
  */
-export async function waitToGetOkPins (cid, cluster, waitTime = MAX_PIN_STATUS_CHECK_TIME, checkInterval = PIN_STATUS_CHECK_INTERVAL) {
+export async function waitOkPins (cid, cluster, waitTime = MAX_PIN_STATUS_CHECK_TIME, checkInterval = PIN_STATUS_CHECK_INTERVAL) {
```

Comment:
Like the default values!

Maybe this could do with some documentation and comments? And some method renaming for clarity?
Would Promise.all() be any use here?

Reply:
Added some documentation, hope it helps. 
Not sure what you have in mind with `Promise.all()`, please feel free to suggest if you have anything specific in mind. Same for a better name for the function.

Reply:
Method comment definitely helps now.
Was just wondering if `Promise.all()` might help or simplify, but probably not in this case since we're just retrying until a time.